consul/server_manager/server_manager.go

```diff
@@ -47,11 +47,19 @@ const (
 	newRebalanceConnsPerSecPerServer = 64
 )
 
+// ConsulClusterInfo is an interface wrapper around serf and prevents a
+// cyclic import dependency
 type ConsulClusterInfo interface {
 	NumNodes() int
 }
 
-// serverCfg is the thread-safe configuration struct used to maintain the
+// ConnPoolTester is an interface wrapping client.ConnPool to prevent a
+// cyclic import dependency
+type ConnPoolPinger interface {
```

Comment:
I'd just call this "Pinger" since it doesn't have anything to do with a conn pool, except as a detail of a specific implementation.